packages/account/src/account.tsx

```diff
@@ -0,0 +1,145 @@
+import { loadWidget } from '@openstad-headless/lib/load-widget';
+import '@utrecht/component-library-css';
+import '@utrecht/design-tokens/dist/root.css';
+import { FormFieldTextbox, Heading, Paragraph, Button } from '@utrecht/component-library-react';
+import React, { useState } from 'react';
+import './account.css';
+import { ProjectSettingProps, BaseProps } from '@openstad-headless/types';
+
+export type AccountWidgetProps = BaseProps &
+  AccountProps &
+  ProjectSettingProps & {
+    resourceId?: string;
+  };
+
+
+
+export type AccountProps = {
+  allowNickname?: boolean;
+  minLength?: number;
+  maxLength?: number;
+  allowUserEdit?: boolean;
+  formData?: object;
+};
+
+function Account({
+  allowNickname = true,
+  minLength = 2,
+  maxLength = 140,
+  allowUserEdit = true,
+  formData = {
+    email: {
+      value: 'test@draad.nl',
+      label: 'E-mailadres',
+    },
+    name: {
+      value: 'John',
+      label: 'Naam',
+    },
+    straatnaam: {
+      value: 'Nieuwstraat',
+      label: 'Straatnaam',
+    },
+    huisnummer: {
+      value: '9',
+      label: 'Huisnummer',
+    },
+    postalCode: {
+      value: '1234AB',
+      label: 'Postcode',
+    },
+    city: {
+      value: 'Nijkerk',
+      label: 'Woonplaats',
+    },
+    nickname: {
+      value: 'johndoe12',
+      label: 'Schermnaam',
+    }
```

Comment:
Moeten hier al `value`s in staan? Want als het leeg kan blijven zou het mooier zijn. Anders wat generiekere defaults gebruiken aub.

Reply:
Heb ze leeg gemaakt nu, het ophalen van de data zit in een andere branch waar nog wat issues mee waren.